kaniko.go

```diff
@@ -264,3 +283,60 @@ func getDigest(digestFile string) string {
 func trace(cmd *exec.Cmd) {
 	fmt.Fprintf(os.Stdout, "+ %s\n", strings.Join(cmd.Args, " "))
 }
+
+func getPluginEnvVars(cmdArgs []string) []string {
+	envVars := os.Environ()
+
+	// Iterate through environment variables
+	for _, envVar := range envVars {
+		// Check if the variable starts with PLUGIN_
+		if strings.HasPrefix(envVar, "PLUGIN_") && !contains(excludeList, envVar) && contains(allowList, envVar) {
```

Comment:
you don't need an exclude list since you already have an `allowList` to only allow certain values in kaniko

Reply:
done